src/js/tabs/send.js

```diff
@@ -792,7 +792,8 @@ SendTab.prototype.angular = function (module)
         tx.submit();
       }
       else {
-        tx.tx_json.Sequence = Number($scope.send.sequence);
+        tx.tx_json.Sequence = Number($scope.sequence);
+        $scope.incrementSequence($scope.sequence);
```

Comment:
You don't need to pass the `$scope.sequence` to the `incrementSequence` function. It already has access to the `$scope.sequence`.